apps/teams-test-app/src/App.tsx

```diff
@@ -68,12 +68,18 @@ import WebStorageAPIs from './components/WebStorageAPIs';
 
 const urlParams = new URLSearchParams(window.location.search);
 
+const getOriginParam =
+  urlParams.has('doesInsecureUrlHasAppIdException') && urlParams.get('doesInsecureUrlHasAppIdException')
```

Comment:
```suggestion
  urlParams.has('doesInsecureUrlHaveAppIdException') && urlParams.get('doesInsecureUrlHaveAppIdException')
```

Reply:
Updated the name to `origins` it sounded better